Modules/Filtering/LabelMap/test/itkLabelImageToLabelMapFilterTest.cxx

```diff
@@ -24,9 +24,13 @@ void
 zeroSizeCase()
 {
   // test filter with zero sized image
-  auto p_filter = itk::LabelImageToLabelMapFilter<itk::Image<unsigned char, 3>>::New();
-  auto p_image = itk::Image<unsigned char, 3>::New();
-  p_image->SetRegions(itk::Size<3>{ 0, 0, 0 });
+  constexpr unsigned int ImageDimension = 3;
+  using PixelType = unsigned char;
+  using ImageType = itk::Image<PixelType, ImageDimension>;
+  auto p_filter = itk::LabelImageToLabelMapFilter<ImageType>::New();
+  auto p_image = ImageType::New();
+
+  p_image->SetRegions(ImageType::SizeType::Filled(0));
```

Comment:
An image region is already empty by default: 

https://github.com/InsightSoftwareConsortium/ITK/blob/v5.3rc01/Modules/Core/Common/include/itkImageRegion.h#L365

So the entire `p_image->SetRegions` call may be removed  😃

Reply:
:+1: Removed in 33d1d0d.